### ISSUE TITLE
Add number, duration, and subject elements to either pbcoreDescription or pbcorePart

### ISSUE THREAD

**Issue by GlennClatworthy** [WGBH/PBCore2.0#68]
Content-oriented segments of a resource may have a segment number and length without a specific start and end time. Categorization may change from segment to segment as well. If pbcorePart is the most appropriate place to define segments, I would recommend adding segment number and length elements. Alternatively, pbcoreDescription would require a subject element as well. See this example using a recent Charlie Rose episode:

```
<pbcoreDescription>
    <pbcoresegmentType>Story</pbcoresegmentType>
    <pbcoresegmentNumber>3</pbcoresegmentNumber>
    <pbcoresegmentLength>00:14:48</pbcoresegmentLength>
    <pbcoreannotation>
        A look at the Morgan Library’s “Marks of Genius” exhibit with curator John McQuillen.
    </pbcoreannotation>
    <pbcoresegmentCategory source={}>Arts</pbcoresegmentCategory>
    <pbcoresegmentCategory source={}>Culture</pbcoresegmentCategory>
    <pbcoresegmentCategory source={}>Media</pbcoresegmentCategory>
</pbcoreDescription>
```

**Comment by AllisonAnn** [comment]
I agree that Duration would be a useful element to associate with segments.  I also think that we should have a general Duration field associated with the resource as a whole, in the Intellectual Content class (see issue #61)

I would like to recommend there be a way however, to indicate that Duration at the Intellectual Content level and/or Segment level is approximate (or actual), since most Instantiations will have slightly (or significantly) different duration times across the board for a given resource.  For this reason and others, I've found that an approximate duration is useful at the Intellectual Content level.
